### ISSUE TITLE
Adding the Zaza language

### ISSUE THREAD

**Issue by CrowdsourceKurdish** [common-voice/common-voice#3820]
We demand that the Zaza language be added to the common project, we will create a community for the project.
Language name: Zaza
Language code: zza

Thanks.
#Kurdish Language Community

**Comment by ftyers** [comment]
Dear @CrowdsourceKurdish thank you for your interest! We look forward to supporting Zaza on Common Voice! Could you please share the following, I have filled in some already:

- Language name (in Zaza): *Zazakî*
- Language code: `zza`
-  Language Speaker Population: 4 million
-  Plural forms: ?
-  Script: Latin

**Comment by CrowdsourceKurdish** [comment]
Hello, I have now discussed this topic with our University teacher. He said it has a two(dual) form through Unicode standards.
![aaa](https://user-images.githubusercontent.com/72891963/194386381-8f7680ef-11a7-4c92-a8e9-1df7ad997f83.jpg)

**Comment by CrowdsourceKurdish** [comment]
Hi,
Hello, when will Zazakî be added?
We are excited to work.

**Comment by CrowdsourceKurdish** [comment]
Hello,
We started collecting sentences last week, we have enough sentences. We worked on Kurdish (kurmanji) before, we collected 95 hours of audio data, and we want to do the same for Zazakî.
Please let us know when Zazakî is added to the pontoon.

Thanks.

**Comment by ftyers** [comment]
Dear @CrowdsourceKurdish, still no update. I will post here when I have further information.

**Comment by CrowdsourceKurdish** [comment]
@ftyers 
Thanks for the information, we look forward to your reply.

**Comment by CrowdsourceKurdish** [comment]
Hi @ftyers 
Thank you very much for this good news.
I've done a few translations on the project and you can do local translator on me.
I also met with other translator friends and we will start translating as soon as possible.
Thank you again.

Kurdish Crowdsource Team

**Comment by CrowdsourceKurdish** [comment]
Hello, @ftyers 
Can you add these two participants as translators for Zazaki?


![aaa](https://user-images.githubusercontent.com/72891963/198831827-f42624c4-10d6-4fed-82de-2ead77ead096.jpg)

**Comment by ftyers** [comment]
Done!

**Comment by CrowdsourceKurdish** [comment]
Yes, they are added as contributors, but translations appear as suggestions. Contributors must be set as translators.

**Comment by ftyers** [comment]
They are translators, if you added suggestions they will need to now be reviewed. They don't automatically change.

**Comment by CrowdsourceKurdish** [comment]
Now I understand the localization process.  You can cast Mahir for the review, Mahir is a linguist and is currently doing a master's in Zazaki.

Thanks.

**Comment by ftyers** [comment]
Done!
